Gem/Code/Source/Components/NetworkPlayerMovementComponent.cpp

```diff
@@ -137,6 +137,12 @@ namespace MultiplayerSample
         // Inputs for your own component always exist
         NetworkPlayerMovementComponentNetworkInput* playerInput = input.FindComponentInput<NetworkPlayerMovementComponentNetworkInput>();
 
+        if (m_toggleSprint)
+        {
+            m_sprinting = !m_sprinting;
+            m_toggleSprint = false;
+        }
+
```

Comment:
Should the toggle move into the "if PlayerActionsAllowed() == All" block below (line 160ish)? It seems a bit weird to toggle when the player can't move...

Reply:
oh.. yup I can make that change 👍